codility_training/lessons.lesson03.TimeComplexity.PermMissingElem/NowonLee.js

```diff
@@ -0,0 +1,17 @@
+function solution(A) {
+    // write your code in JavaScript (Node.js 8.9.4)
+    var r = 0
+    A.sort(function(a,b){return a-b})
+    if(A[0] !== 1) return 1;
+    else if(A[A.length-1]  !== A.length+1) return A.length+1;

```

Comment:
```suggestion
    const len = A.length
    if(A[len - 1]  !== len+1) return len+1;
```
- length를 계속 호출하면 계산 비용이 소모됨
- `else`는 사용할 필요 없음